ts/store/reducers/entities/services/servicesByOrganizationFiscalCode.ts

```diff
@@ -55,28 +57,31 @@ export function serviceIdsByOrganizationFiscalCodeReducer(
       const serviceTuples = action.payload;
```

Comment:
```suggestion
      const serviceTuples: ReadonlyArray<ITuple2<string, string | undefined>> = action.payload;
```
to avoid any type being associated to `serviceTuples`